tools/table_annovar/table_annovar.xml

```diff
@@ -34,13 +39,23 @@
         #end if
 
         #set protocol = ','.join( $protocol )
-        #set operation = ','.join( $operation ) 
+        #set operation = ','.join( $operation )
+
+        ## Annotate variants.
+        table_annovar.pl ${tab_anno_input} ${__get_data_table_entry__('annovar_indexes', 'dbkey', $input.dbkey, 'path')} -protocol ${protocol} -operation ${operation} -nastring '.' -buildver ${input.dbkey} --outfile output
 
-        ## Reduce variants.
-        ; table_annovar.pl input.avinput ${__get_data_table_entry__('annovar_indexes', 'dbkey', $input.dbkey, 'path')} -protocol ${protocol} -operation ${operation} -nastring NA -buildver ${input.dbkey} --outfile output
+        ## Add option to consume/produce VCF.
+        #if str($out_format) == "vcf":
+            --vcfinput
```

Comment:
I'm not familiar with this tool, but is `--vcfinput` correct, not `--vcfoutput`?